pkgs/servers/mail/dovecot/default.nix

```
@@ -61,6 +76,9 @@ stdenv.mkDerivation rec {
       libcap
       inotify-tools
     ]
+    ++ lib.optional (stdenv.hostPlatform.isLinux && !stdenv.hostPlatform.isDarwin) libtirpc
+    ++ lib.optional (stdenv.hostPlatform.isLinux && withApparmor) libapparmor
```

```suggestion
    ++ lib.optional withApparmor libapparmor
```
we normally do this to explicitly honor the users override

---

Thanks, fixed.